superset/connectors/sqla/models.py

```diff
@@ -19,7 +19,7 @@
 import re
 from collections import OrderedDict
 from datetime import datetime
-from typing import Any, Dict, List, NamedTuple, Optional, Tuple, Union
+from typing import Any, Dict, Hashable, List, NamedTuple, Optional, Tuple, Union
```

Comment:
Might have missed it, but are there any `Hashable`s that aren't `Dict`s?

Reply:
In this case the `Hashable`s are a list of objects added via `cache_key_wrapper`. Come to think of it, it might be a good idea to check that the objects added via `cache_key_wrapper` are in fact hashable and throw an understandable error if such is not the case. However, will leave that to another PR.